maven-plugin-plugin/pom.xml

```diff
@@ -271,7 +271,7 @@
     <plugins>
       <plugin>
         <groupId>org.apache.maven.plugins</groupId>
-        <artifactId>maven-plugin-plugin</artifactId>
+        <artifactId>maven-plugin-report-plugin</artifactId>
```

Comment:
Here we will have a cycle ... `maven-plugin-report-plugin` also requires `maven-plugin-plugin`